tests/components/electric_kiwi/test_sensor.py

```diff
@@ -65,6 +69,50 @@ async def test_hop_sensors(
     assert state.attributes.get(ATTR_DEVICE_CLASS) == SensorDeviceClass.TIMESTAMP
 
 
+@pytest.mark.parametrize(
+    ("sensor", "sensor_state"),
+    [
+        ("sensor.total_running_balance", "184.09"),
+        ("sensor.total_account_balance", "-102.22"),
+        ("sensor.next_billing_date", "2020-11-03T00:00:00"),
+        ("sensor.hop_percentage", "3.5"),
```

Comment:
```suggestion
        ("sensor.total_running_balance", "184.09"),
        ("sensor.total_current_balance", "-102.22"),
        ("sensor.next_billing_date", "2020-11-03T00:00:00"),
        ("sensor.hour_of_power_savings", "3.5"),
```

Reply:
With that change it now use the right entity id I think and thereby the tests works